<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSaveWgt</class>
 <widget class="QWidget" name="MainSaveWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="m_pBtnAddItem">
        <property name="text">
         <string>增加新事项</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pBtnSaveAll">
        <property name="text">
         <string>保存所有</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pBtnExport">
        <property name="text">
         <string>导出存档</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pBtnImport">
        <property name="text">
         <string>导入存档</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pCBFinished">
        <property name="text">
         <string>显示已完成</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pCBUnfinished">
        <property name="text">
         <string>显示未完成</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="m_pTabWgt"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
